src/UI/NavigationContainer.elm

```diff
@@ -145,7 +147,8 @@ type alias Content msg =
 See [`Nav.dialog`](#dialog) to see how to create a dialog.
 -}
 type Dialog msg
-    = Dialog (Dialog.Dialog msg)
+    = Dialog1 (Dialog1.Dialog msg)
```

Comment:
You don't need this. It could be:
```
type alias Dialog msg =
   (Dialog2.Dialog ms)
```
The dialog v1 can be described as a v2.

Reply:
I do prefer @segmentationfaulter version, we don't want to replace the old version, we want to support both. so the old version should be available.